From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect struct sdhci_host
Patch-mainline: never, kabi

In 4.4.21, commit b07ef1af4d0e7130ac1917f662b1e410e9b09555 (mmc:
sdhci: 64-bit DMA actually has 4-byte alignment), upstream commit
04a5ae6fdd018af29675eb8b6c2550c87f471570 removed 2 members from struct
sdhci_host. It indeed changed the layout of that structure.

Just reintroduce the two members as dummies and make everyone happy,
including the kABI checker. Note that nobody should touch them, they
were only internal variables.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/mmc/host/sdhci.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/mmc/host/sdhci.h
+++ b/drivers/mmc/host/sdhci.h
@@ -488,6 +488,8 @@ struct sdhci_host {
 	dma_addr_t align_addr;	/* Mapped bounce buffer */
 
 	unsigned int desc_sz;	/* ADMA descriptor size */
+	unsigned int align_sz;  /* unused -- kABI */
+	unsigned int align_mask;/* unused -- kABI */
 
 	struct tasklet_struct finish_tasklet;	/* Tasklet structures */
 
